build-system/server.js

```diff
@@ -60,6 +60,7 @@ function proxyToAmpProxy(req, res, minify) {
   request(url, function (error, response, body) {
     body = body
         // <base> href pointing to the proxy, so that images, etc. still work.
+        .replace(/rtv\/\d+\//g, '') // versioned URLs.
```

Comment:
Should we be a bit more specific here? `/https:\/\/cdn.ampproject.org\/rtv\/\d+\/`